src/contacts/EditContact/EditContact.js

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import stripesForm from '@folio/stripes/form';
+import {
+  Accordion,
+  AccordionSet,
+  Col,
+  Row,
+  Select,
+  TextField,
+} from '@folio/stripes/components';
+
+import AddressForm from './AddressForm';
+import EmailForm from './EmailForm';
+import PhoneForm from './PhoneForm';
+import UrlForm from './UrlForm';
+import css from './EditContact.css';
+
+class EditContact extends Component {
+  static propTypes = {
+    stripes: PropTypes.object.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    change: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const { stripes: { store }, change, dispatch } = this.props;
+
+    return (
+      <AccordionSet>
+        <Accordion
```

Comment:
do we really need accordion? it was on mock up, but I think Dennis just pasted it from somewhere

Reply:
I don't know. Let's leave it as on mock-up now

Reply:
FYI Dennis closed my ticket without accordion